package.json

```diff
@@ -36,6 +36,7 @@
   },
   "devDependencies": {
     "@multiversx/sdk-network-providers": "1.2.1",
+    "@multiversx/sdk-network-providers-next": "npm:@multiversx/sdk-network-providers@1.6.0-beta.0",
```

Comment:
All right for now (to reference the beta version), since it's used for tests only (dev dependency). Once `sdk-core` is compatible with both the old and the new `sdk-network-providers` (as of this PR), we can make a proper release there, as well. And afterwards, come back here to adjust the reference.

For `sdk-core`, let's bump de minor version.

Reply:
Ok, got it. So after this PR is merged we make a proper release for sdk-network-providers, then come back here and reference the proper release. Will also make a new release for sdk-core increasing the minor version. 👍🏻